recipe/meta.yaml

```diff
@@ -18,28 +17,39 @@ source:
     # into various problems around ABIs of numpy/C++ etc.
     # Hopefully, we can provide more specific builds per CPU architecture
     # in the future, see conda-forge/conda-forge.github.io#1261
-    - patches/0002-silence-log-that-tells-people-to-install-from-pip.patch
+    - patches/0001-silence-log-that-tells-people-to-install-from-pip.patch
```

Comment:
I once heard (and did myself) that it's always a good idea to be eloquent about the origin of the patches. If the comment above applies, ignore this comment. I would also reference the PRs or GH issues that originate the patches in the PR description, but maybe that's just me.

Reply:
I am unable to find all the PRs or GH issues that mention the fixes covered by patches. Four patches were already there, added by CF. I was unable to build without these.